kafka_utils/kafka_check/main.py

```diff
@@ -106,6 +117,9 @@ def run():
     # to prevent flooding for sensu-check.
     logging.getLogger('kafka').setLevel(logging.CRITICAL)
 
+    if (args.controller_only or args.first_broker_only) and args.broker_id is None:
```

Comment:
Probably these options controller_only and first_broker_only should be mutually exclusive. We can add check for that.